packages/table/src/docs/table-api.md

```diff
@@ -33,6 +33,8 @@ type ICellMapper<T> = (rowIndex: number, columnIndex: number) => T;
 
 @method Table.scrollToRegion
 
+@method Table.scrollByOffset
```

Comment:
I think you can avoid adding this method to Table and only use Table2:

```suggestion
@method Table2.scrollByOffset
```

Reply:
*facepalm* that makes sense

Reply:
for some reason I thought it had to be on table 1